pkg/cmd/container/run_runtime.go

```diff
@@ -25,48 +25,42 @@ import (
 	"github.com/containerd/containerd/oci"
 	"github.com/containerd/containerd/plugin"
 	runcoptions "github.com/containerd/containerd/runtime/v2/runc/options"
-	"github.com/containerd/nerdctl/pkg/api/types"
-	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
 )
 
-func generateRuntimeCOpts(cmd *cobra.Command, globalOptions types.GlobalCommandOptions) ([]containerd.NewContainerOpts, error) {
-	runtime := plugin.RuntimeRuncV2
+func generateRuntimeCOpts(cgroupManager, runtime string) ([]containerd.NewContainerOpts, error) {
+	runTime := plugin.RuntimeRuncV2
```

Comment:
Can we change the `runtime` arg to `runtimeStr`, and `runTime` to `runtime`?

It's more consistent with existing format and having a `Str` suffix is more clear than an uppercase `T`